modules/ObjectControl/src/objectcontrol.cpp

```diff
@@ -351,6 +351,7 @@ void ObjectControl::loadScenario() {
 	// TODO query scenario participants from scenario manager
 	auto request = std::make_shared<atos_interfaces::srv::GetObjectIds::Request>();
 	auto promise = idClient->async_send_request(request, idsCallback);
+	promise.wait_for(std::chrono::seconds(10));
```

Comment:
I have just started testing, but when I press `init` it takes 10 seconds for ATOS to get to the initialised state and I think that this is why. When you do this I think you block the thread for 10 seconds, which isn't what we want to do